main_test.go

```diff
@@ -161,7 +161,14 @@ func getTestApp(t *testing.T, cfg *configs.Config, ignoreLeaks bool) TestApp {
 
 	km := basic.NewKeyManager(cfg, keyStore, fc)
 
-	wp := jobs.NewWorkerPool(jobStore, 5, 1)
+	wp := jobs.NewWorkerPool(
+		jobStore, 100, 1,
```

Comment:
Note; the pool capacity affects the `async create exceeding worker pool capacity` test case also.